docker-compose.test.yml

```diff
@@ -9,42 +9,7 @@ services:
       - ./shared_files/decrypted/local.env
     environment:
       - DB=postgresql://postgres:toor@db-unit-test:5432
-      - ATO_PUBLIC_KEY_FILE=../../shared_files/ATO_public.pem
```

Comment:
Env vars folded into the local.env. This allows us to leverage a single env file for use in vscode.